### DISCUSSION TITLE
pyMOR Community Meeting 2023-08-22

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#2139] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, August 22, at 14:00 CEST and 08:00 EDT, online at:

Link: https://wwu.zoom-x.de/j/61946685491?pwd=ZFNKeW1Gbk40VWl2OTdaZldqYkl6Zz09

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] pyMOR school program (@pmli)
- [x] pre-commit (@pmli)
- [x] docs deployment (@sdrave)
- [x] gram_schmidt performance (@sdrave)

Moderator: @pmli

**Comment by lbalicki** [comment]
Meeting Notes:
- pre-commit
  - main developers voted to enforce usage of pre-commit for linting beyond Python and Markdown (e.g., toml, yaml, bibtex) as long as it doesn't take too long
  - we still need to discuss which linter to use and for which file formats good linters even exists
  - it would be nice if the linters allow for choosing different configuration options
- docs deployment
  - documentation is very large, github pages doesn't support our docs size
  - will look into other options (different places for deployment, reducing visualization or problem sizes in tutorials, ...)
- gram_schmidt performance
  - there are several ways to proceed
  - would be interesting to compare whether underlying algorithm or BLAS 1/2 vs BLAS 3 or abstraction is effecting performance the most

Next Meeting: 09-11-2023 at 14:00 CEST and 08:00 EDT
Moderator: Linus
